models/unifiedparticletransformer_AK4_V01/1/modelfile

```diff
@@ -0,0 +1 @@
+modelfile
```

Comment:
@y19y19 this caused the unit test error in https://github.com/cms-sw/cmssw/pull/47702: this file is currently a symbolic link to itself